tiles/rooms/wall/ColorWallTop.svg

```diff
@@ -0,0 +1,15 @@
9LTeCwCCCAgG8FCF7zPHSkCOcZnMMhgEAgBUgbzt+wkiKcP2uOhAACARfgzGseBlhnW89rFuEtpAjnAZtDIIBAUwgQvBo7zHs0GeMWTcZ4obGHoXYEEECguQRIGzZmvCdThFqL8EKdbRG4GmNMrQgg0MQCnHnVefBJEdYZlOoQQACBaQQIXtOg1PgWKcIa4SiGAAIIVCtA8KpW7Mj9s7qudY8mZGzU40qcdCEvBBBAAIEGCxC8zICfLZVKt61Zs+aAWTWURgABBBCoRoDgVY3WoX11Xavj0AK6L9VQnCIIIIAAAoYCBK/qACdThIsWLXqgra3NeXQJLwQQQAABFwQIXnNHJ0U4dyv2RAABBBoqQPCahZcU4SxAbEYAAQRcECB4zYw+rlmEdylF+HtShDMjsQUBBBBwQ4DgdaS6rbee1oSMO1auXJk4cjPvIIAAAgi4LUDwmjICTopQZ1trtRbhlilv8yMCCCCAgMcECF7vDAgpQo99MGkOAgggUEmg2YMXKcJKnw62IYAAAh4VaObgtU1jso4UoUc/mTQLAQQQqCDQdMFL17WG5XGngtaTur5VrmDDJgQQQAABjwo0zfO8FLScQLXJsqzzVq9e/QSBy6OfSJqFAAIIzEGgWc68til4rVXQ2j4HE3ZBAAEEEPC4QKCDFylCj3/6aB4CCCBQo0Ag04akCGv8NFAMAQQQ8IlAEM+8Nit4rSNF6JNPIM1EAAEEahAIUvBKKGjdrlmEz2gyhnP/Fi8EEEAAgYAKBCF4Oc/VerS1tXX9ihUrxgM6TnQLAQQQQGCKgN+D12ZnLcJVq1ZFpvSJHxFAAAEEAi7g1+BFijDgH0y6hwACCFQS8FvwIkVYaTTZhgACCDSJgG+ClyZjvKhnbK0jRdgkn0y6iQACCFQQ8HzwUtA6oOtat2rq+3MV+sEmBBBAAIEmEvDyTcpOivBhBa7zNf2dwNVEH0q6igACCMwm4MkzL1KEsw0b2xFAAIHmFvBU8CJF2NwfRnqPAAIIzFXAK2nDvBq8gRThXIeN/RBAAIHmFnD9zEtnW89rFuEtmkW4t7mHgt4jgAACCMxVwM3gtUdnWrdoMsYLc20s+yGAAAIIIOAIuJE2nEwRai3CC3W2ReDic4gAAgggULXAvJ55kSKsenwogAACCCAwjcB8BS9ShNPg8xYCCCCAQG0CjQ5eWV3XukcTMjbqcSVOupAXAggggAACxgKNDF7Plkql29asWXPAuJVUgAACCCCAwBSBugcvXdfqOLSA7ktTjsOPCCCAAAII1E2gZfHixVnVFp1DjeFZ9nFShHeqvr/RLEIC1yxYbEYAAQQQqF1ggQJOVmdLH06lUj/S33fp9yXTVaf3KwUvUoTTofEeAggggEBDBCbThopLkyu4K4A9owD2oI70Nf2pFKwmG0OKsCFjQqUIIIAAArMIHHaTslJ+A0uWLLlRQekyldtZoey4At6PSRFWEGITAggggEDDBKadsHHCCSdsVQC7OJ1Of6dcLt+jQHXioRbY+vtpTci4Y+XKlYmGtYqKEUAAAQQQMBEYGxs7UX8e37p16389//zzl5vURVkEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAAAEEEEAAAQQQQAABBBBAwCcC/wd4S3JjNvd7DQAAAABJRU5ErkJggg=="></image>
```

Comment:
As I am looking at your rectangle, I would like to know what is behind it what is in this image?